src/hu/elte/txtuml/stdlib/util/regex/MatcherImplementation.java

```diff
@@ -0,0 +1,273 @@
+package RegularExpression;
+
+public class MatcherImplementation implements  Matcher{
+	
+	java.util.regex.Matcher matcher;
+	
+	public MatcherImplementation(java.util.regex.Matcher matcher){
+		this.matcher = matcher;
+	}
+	public MatcherImplementation(){
```

Comment:
I propose to merge the two constructors. See the comment at the previous constructor.